### ISSUE TITLE
Catch file extension in SketchupExtension.new

### ISSUE THREAD

**Issue by thomthom** [SketchUp/rubocop-sketchup#28]
Often people hard code file extension into SketchupExtension.new.
Catch and warn against this.

**Comment by thomthom** [comment]
Currently blocked by error being raised in one of the matchers. Added comment in RuboCop's issue tracker to what seemed to be relevant issue; https://github.com/bbatsov/rubocop/issues/5163#issuecomment-347996981

**Comment by thomthom** [comment]
Logged as new issue - as the previous one was trying to capture node types (different scenario).

https://github.com/bbatsov/rubocop/issues/5656
